molecule/packages-dynamic/molecule.yml

```diff
@@ -1,7 +1,7 @@
 ---
 dependency:
   name: galaxy
-  role-file: requirements.yml
+  requirements-file: requirements.yml
```

Comment:
given my observation, this will not work with the last version of molecule released for python2.